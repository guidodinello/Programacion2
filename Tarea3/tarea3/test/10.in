# Prueba removerDeBinario, esAvl, alturaBinario, cantidadBinario, imprimirBinario, insertarEnBinario
insertarEnBinario (10,10.0)
insertarEnBinario (5,5.0)
insertarEnBinario (20,20.0)
insertarEnBinario (25,25.0)
insertarEnBinario (15,15.0)
insertarEnBinario (7,7.0)
insertarEnBinario (2,2.0)
imprimirBinario
esAvl
cantidadBinario
alturaBinario
removerDeBinario 2 
insertarEnBinario (2,2.0)
removerDeBinario 7 
insertarEnBinario (7,7.0)
removerDeBinario 15 
insertarEnBinario (15,15.0)
removerDeBinario 25 
insertarEnBinario (25,25.0)
removerDeBinario 5 
removerDeBinario 20 
insertarEnBinario (1,1.0)
insertarEnBinario (12,12.0)
removerDeBinario 10 
removerDeBinario 7 
removerDeBinario 2 
removerDeBinario 1 
removerDeBinario 15 
removerDeBinario 12 
removerDeBinario 25 
esAvl
imprimirBinario
cantidadBinario
alturaBinario
Fin