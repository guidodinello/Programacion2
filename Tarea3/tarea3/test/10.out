1>#  Prueba removerDeBinario, esAvl, alturaBinario, cantidadBinario, imprimirBinario, insertarEnBinario.
2>Insertado.
3>Insertado.
4>Insertado.
5>Insertado.
6>Insertado.
7>Insertado.
8>Insertado.
9>
--(25,25.00)
-(20,20.00)
--(15,15.00)
(10,10.00)
--(7,7.00)
-(5,5.00)
--(2,2.00)
10>AVL.
11>7.
12>3.
13>Removido.
14>Insertado.
15>Removido.
16>Insertado.
17>Removido.
18>Insertado.
19>Removido.
20>Insertado.
21>Removido.
22>Removido.
23>Insertado.
24>Insertado.
25>Removido.
26>Removido.
27>Removido.
28>Removido.
29>Removido.
30>Removido.
31>Removido.
32>AVL.
33>
34>0.
35>0.
36>Fin.
